src/Microsoft.AspNetCore.Server.Kestrel.Https/HttpsConnectionAdapter.cs

```diff
@@ -105,28 +105,6 @@ public HttpsConnectionAdapter(HttpsConnectionAdapterOptions options, ILoggerFact
             return new HttpsAdaptedConnection(sslStream);
         }
 
-        private static X509Certificate2 ConvertToX509Certificate2(X509Certificate certificate)
-        {
-            if (certificate == null)
-            {
-                return null;
-            }
-
-            X509Certificate2 certificate2 = certificate as X509Certificate2;
-            if (certificate2 != null)
-            {
-                return certificate2;
-            }
-
-#if NETSTANDARD1_3
-            // conversion X509Certificate to X509Certificate2 not supported
-            // https://github.com/dotnet/corefx/issues/4510
-            return null;
-#else
-            return new X509Certificate2(certificate);
```

Comment:
No. It's also no longer necessary.

https://github.com/aspnet/KestrelHttpServer/pull/1810#discussion_r114911982